### ISSUE TITLE
Implement CommonController: 3h

### ISSUE THREAD

**Issue by VladOSIO2** [CalculusEnjoyer/intellistart-java-2022-vanquishers#38]
Implement MainController class for managing next main, 'any user can request' endpoints:

- getting the current week number
```
GET /weeks/current

Response:
{
    "weekNum": 11
}
```

- getting the next week number
```
GET /weeks/next

Response:
{
    "weekNum": 12
}
```

- getting information about the current user
```
GET /me

Response:
{
    "email": "your@email.com",
    "role": "INTERVIEWER",
    "id": "some-UUID-only-for-COORDINATOR-or-INTERVIEWER"
}
```

For the Guest, response should be same as for any other endpoint, not allowed for Guests:

```
HTTP 403
{
    "errorCode": "not_authorized",
    "errorMessage": "You are not authorized to use this functionality"
}
```

**Comment by VladOSIO2** [comment]
Implemented endpoint method templates (PR #52), Implemented current week & next week endpoints (PR #59)

**Comment by VladOSIO2** [comment]
Implementation of /me was moved to a different issue (#101)
